z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-installer-client/tasks/main.yaml

```diff
@@ -10,9 +10,19 @@
 ---
 # tasks file for configure-installer-and-image
 
+- name: Verify openshift minor version
+  shell: |
+    version=$(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{{ openshift_version }}.{{ openshift_minor_version }}/release.txt | grep 'Name:' | awk -F: '{print $2}' | xargs)
```

Comment:
The link here should also involve {ansible_architecture}, such as:
https://mirror.openshift.com/pub/openshift-v4/s390x/clients/ocp/4.11.10/release.txt

Reply:
sure, I forget to add the {{ ansible_architecture }}.